File: two-sum/two_sum.cpp
#include <iostream>
#include <vector>

class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        
        std::vector result;
        for (int i = 0; i < nums.size(); i ++) {
            int test = nums[i];
            for (int j = i+1; j < nums.size(); j ++) {
                int check = test + nums[j];
                if (check == target) {
                    result.push_back(nums[i]);
                    result.push_back(nums[j]);
                }
            }
        }
        return result;
    }
};

File: array/two_sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        result = []
        for i in range(len(nums)):
            if len(result) == 2:
                return result
            for j in range(len(nums)):
                if nums[i] + nums[j] == target:
                    result.append(i)
                    result.append(j)
        return result